x-pack/plugins/observability/public/pages/overview/containers/overview_page/overview_page.tsx

```diff
@@ -276,17 +279,13 @@ function PageHeader({
 
   return (
     <EuiFlexGroup wrap gutterSize="s" justifyContent="flexEnd">
-      <EuiFlexItem grow={1}>
-        <EuiTitle>
-          <h1 className="eui-textNoWrap">{overviewPageTitle}</h1>
-        </EuiTitle>
-      </EuiFlexItem>
       <EuiFlexItem grow={false}>
         <DatePicker
           rangeFrom={relativeStart}
           rangeTo={relativeEnd}
           refreshInterval={refreshInterval}
           refreshPaused={refreshPaused}
+          width="auto"
```

Comment:
What is this change about?

Reply:
Is this change intentional?

| Before | After |
|--------|-------|
|![image](https://user-images.githubusercontent.com/12370520/203768409-35e23dce-f2b4-4389-9932-b49f77008d30.png)|![image](https://user-images.githubusercontent.com/12370520/203768437-d2c16abb-82de-405f-a90f-fe59844f8e2e.png)|

Reply:
I think the new Unified Search bar also employs this to make the size of the date picker component dynamic to the displayed value or label. I'm fine with moving in this direction, and we'll probably have to pick this up in other apps later on (if they're not already moving to the new Unified Search bar).

Reply:
Interesting, may I ask what was the reasoning behind it?

Reply:
@maryam-saeidi To save space in context of its surrounding elements, for instance the Unified Search bar date picker changes width depending on the value or content within.

<img width="1663" alt="image" src="https://user-images.githubusercontent.com/4104278/203774651-c1cb36bf-e757-41ce-8e8c-7bc395805f19.png">

<img width="1664" alt="CleanShot 2022-11-24 at 12 33 49@2x" src="https://user-images.githubusercontent.com/4104278/203774635-792a864f-3d73-4526-8422-eb7b5b8b276d.png">

The previous date picker in the Observability overview page had a fixed size which was the largest width possible (when the date range would be absolute date formats) which is unnecessary for most use-cases.